### DISCUSSION TITLE
Why I’ve joined this program

### DISCUSSION THREAD

**Discussion by maryonique** [enkidevs/curriculum#3120] [General]
I’ve joined this program due to finally finding my passion! It may come easy but I am ready to be a eager student and learn all there is to learn about coding!

**Comment by nem035** [comment]
Welcome @maryonique, it's a pleasure having you. 

Good luck on your learning journey 🚀 🙌

**Comment by chwa982** [comment]
Similar story here and loving Enki so far! Great design—simple and fun.

**Comment by nem035** [comment]
Thanks @chwa982, we appreciate it :)

**Comment by BigOn1110** [comment]
I also want to thanks the team.

**Comment by Ceridwyn2** [comment]
I’m a relative newbie to programming languages. I have some experience in coding html but it’s been years since I’ve used it in practice. I work in health care (nurse) but I’ve long had an interest in computers.

**Reply by nem035** [reply]
Welcome @Ceridwyn2 :)

**Comment by digibyte69** [comment]
I wanna join how do I go about it?
